exercises/nucleotide-count/test/Tests.hs

```diff
@@ -13,7 +14,9 @@ main = hspecWith defaultConfig {configFastFail = True} specs
 specs :: Spec
 specs = do
 
-          let x `matchesMap` y = x `shouldBe` (Right . fromList) y
+          let x `matchesMap` y = shouldSatisfy x $ \case
+                Right count -> and [ findWithDefault 0 n count == c | (n,c) <- y ]
```

Comment:
I ran `hlint` locally and didn't get any hints/warnings/complaints for this as is. Are there settings for stricter and harsher linting? 

I'd be totally fine with adapting to either suggestion you made in https://github.com/jitwit/haskell/pull/1/files. I am a fan of list comprehensions in general (and I think the combination of getting `count` in scope with pattern matching with comprehension is quite natural), so I'd be happy keeping as is, though I agree that the `and`  is slightly awkward...

Reply:
I thought it might have been [the .hlint.yaml of Kowainik's relude](https://github.com/kowainik/relude/blob/master/.hlint.yaml), but I can't point at the particular rule, so it might have been a different one. (This .hlint.yaml assumes the existence of various relude-specific definitions.)

This solution is quite fine!